site/src/content/docs/docs/integrations/api/graphql.mdx

```diff
@@ -0,0 +1,216 @@
+  v1_transfers(
+    limit: 10,
+    order_by: { source_timestamp: desc }
+  ) {
+    tokens {
+      denom
+      amount
+    }
+	  sender
+	  receiver
+  }
+}
+```
+
+It provides information on the sender, receiver, current stage of the transfer, and asset metadata.
+
+To query for specific addresses, add the `sender` and `receiver` to the where clause:
+
+```graphql
+query GetAllUserTransfers @cached(ttl: 1) {
+  v1_transfers(
+    order_by: { source_timestamp: desc }, 
+    where: {
+      _or: [
+        { sender: { _eq: "union1sw44hacrewvkuvm70zfxhwa9u6vm4f8yc8spqu" }},  
+        { receiver: { _eq: "union1sw44hacrewvkuvm70zfxhwa9u6vm4f8yc8spqu" }}
+      ]
+    }
+  ) {
+    tokens {
+      denom
+      amount
+    }
+    receiver
+    sender
+    source_chain { display_name }
+    destination_chain { display_name }
+  }
+}
+
+```
+Aggregate statistics about all transfers and packets are also available, such as the total amount of transfers and packets:
+
+```graphql
+query {
+ 	v1_statistics {
+    name
+    value
+  }
+}
+```
+
+### Packets
+
+Packets, used in general message passing, can be tracked using v1_packets, similar to v1_transfers.
+
+```graphql
+query GetLatest10Packets @cached(ttl: 1) {
+  v1_packets(
+    limit: 10,
+    order_by: { source_timestamp: desc }
+  ) {
+    source_chain { display_name }
+    destination_chain { display_name }
+		source_data
+		source_sequence
+    destination_sequence
+  }
+}
+```
+
+### Traces
+
+Both transfers and packets come with traces to track the progress of the message. There are four different trace types:
+
+- `SEND_PACKET`
+- `LIGHTCLIENT_UPDATE`
+- `RECEIVE_PACKET`
+- `ACKNOWLEDGE_PACKET`
+
+For a specific transaction hash, you can obtain all traces of transfers and packets caused by this transaction (the `initiating_transaction_hash`) by querying
+
+```graphql
+query GetTracesByHash @cached(ttl: 1) {
+  v1_traces(
+    where: {
+      initiating_transaction_hash: {
+        _eq: "91C934788FD13D5325F5A1D9CF1634D1861E318D6DDD2C141E91CBE8A68A053D"
+      }
+    }
+  ) {
+    type
+    data
+	  transaction_hash
+    chain { display_name }
+  }
+}
+```
+
+The `v1_traces.data` field is dependent on the type and may contain chain-specific information as well. The `v1_transfers` and `v1_packets` 
+also have a relation to traces, to allow for all info of a specific transfer or packet.
+
+```graphql
+query GetLatest10TransfersWithTraces @cached(ttl: 1) {
+  v1_transfers(
+    limit: 10,
+    order_by: {
+      source_timestamp: desc
+    }
+  ) {
+    tokens {
+      denom
+      amount
+    }
+    receiver
+    sender
+    traces {
+      type
+      data
+      transaction_hash
+      chain { display_name }
+    }
+  }
+}
+```
+
+### Assets
+
+Unionlabs provides a curated registry of different assets across different chains, including the bridged asset info for Union's assets. 
+
+```graphql
+query GetAllSupportedAssets @cached(ttl: 1) {
+  v1_assets {
```

Comment:
I recommend inverting this query; make chains toplevel, and then nest assets inside of them. Seeing them grouped by chain makes more intuitive sense